src/inc/til/point.h

```diff
@@ -229,6 +240,19 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             return wil::str_printf<std::wstring>(L"(X:%d, Y:%d)", x, y);
         }
     };
+
+    constexpr point wrap_coord(const COORD rect) noexcept
+    {
+        return { rect.X, rect.Y };

```

Comment:
😢 you said we would migrate to lowercase in the future, but then your new code used uppercase!

Reply:
I'm doing a batch rename of `s/(\.|->)(X|Y|Left|Top|Right|Bottom)/$1\L$2/g` throughout the entire project at the end, after I've migrated everything (`\L` toggles lowercasing in GNU sed for the rest of the expression).
In this particular case however I'm using uppercase X/Y because that's a `COORD`.